monitor/service.go

```diff
@@ -117,6 +117,9 @@ func (m *Monitor) Open() error {
 
 	// If enabled, record stats in a InfluxDB system.
 	if m.storeEnabled {
+		hostname, _ := os.Hostname()
+		m.SetGlobalTag("hostname", hostname)
```

Comment:
`hostname` was previously being set in the `storeStatistics` function, which is called on a separate goroutine. I moved it here to remove a race condition that it may not be set before `WritePoints` is called if the goroutine is not running.